### PR TITLE
Fix turret errors

### PR DESCRIPTION
**Pull request by Koud-Wind**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [ ] ✨ Enhancement
- [ ] 📝 Documentation

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Koud-Wind** [comment]
Please hurry up

**Comment by Desoroxxx** [comment]
> Please hurry up

Okay I get it it's annoying to wait, but it's better for us to take our time and actually test and verify things than to just merge stuff as fast as possible and worsen the current spaghetti situation.

So will test this when I can to see what is happening.

**Comment by Koud-Wind** [comment]
I have tested it, and it is all right.

**Comment by Koud-Wind** [comment]
Added the clear GUI button, perhaps this is https://github.com/Cubed-Development/Modern-Warfare-Cubed/issues/156 Solution, https://github.com/Cubed-Development/Modern-Warfare-Cubed/issues/157 Possible error based on the former

File: src/main/java/com/paneedah/weaponlib/inventory/CustomPlayerInventoryGuiContainer.java [hunk 1]
```diff
@@ -22,6 +22,7 @@ public CustomPlayerInventoryGuiContainer(EntityPlayer player, InventoryPlayer in
     @Override
     public void initGui() {
         super.initGui();
+        this.buttonList.clear();
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/weaponlib/inventory/CustomPlayerInventoryGuiContainer.java:25]: 1. unnecessary
2. Please do not touch anything related to the equipment system as it's being reworked in #138

Comment (Koud-Wind) [reply src/main/java/com/paneedah/weaponlib/inventory/CustomPlayerInventoryGuiContainer.java]: This is a statement that was once included, you may have forgotten to add it

**Review by Desoroxxx — APPROVED** [review]
GitHub makes it hard to see but seems fine to me

**Comment by Desoroxxx** [comment]
# Please from now on use branches, not a fork of yours, on this repo create a new branch like "Turret-Fix" and do your changes,  then once the pr has been merged you can delete that branch, it makes it cleaner and easier to test/work with.

**Comment by Desoroxxx** [comment]
# :warning: **Important Notice**

For enhanced workflow efficiency, please adhere to the following:

1. Use branches, not forks, within this repository.
2. Initiate modifications on a new branch (e.g., "Turret-Fix").
3. Once your pull request is merged, delete the branch.

Following these steps not only tidies the repository, but also accelerates the review process. Thank you!

**Comment by Koud-Wind** [comment]
Please close this issue, https://github.com/Cubed-Development/Modern-Warfare-Cubed/issues/103

**Pull request merged**